contents/tutorials/cross-domain-tracking.md

```diff
@@ -35,29 +35,29 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 import { usePostHog } from 'posthog-js/react'
 
-function FirstDomain() {
+function FirstSite() {
   const posthog = usePostHog()
 
   const sessionId = posthog.get_session_id()
   const distinctId = posthog.get_distinct_id()
 
   return (
     <div className="card">
-      <h1>First Domain</h1>
-      <p>This is the first domain.</p>
-      <Link to={`domain2.com#session_id=${sessionId}&distinct_id=${distinctId}`}>Go to Second Domain</Link>
+      <h1>First Site</h1>
+      <p>This is the first site.</p>
+      <Link to={`domain2.com#session_id=${sessionId}&distinct_id=${distinctId}`}>Go to Second Site</Link>
```

Comment:
Just a matter of curiosity: why `#` instead of, say, query params?

Reply:
No idea 🤷 , if you think its better to do it the other way, I will swap it